api/src/opentrons/protocol_api/module_contexts.py

```diff
@@ -814,103 +814,92 @@ def __init__(
     def target_temperature(self) -> Optional[float]:
-        """Target temperature of the heater-shaker's plate."""
+        """The target temperature of the Heater-Shaker's plate in °C, or ``None`` if no target has been set."""
         return self._module.target_temperature
 
     @property  # type: ignore[misc]
     @requires_version(2, 13)
     def current_temperature(self) -> float:
-        """Current temperature of the heater-shaker's plate."""
+        """The current temperature of the Heater-Shaker's plate in °C. Returns ``23`` in simulation if no target temperature has been set."""
         return self._module.temperature
 
     @property  # type: ignore[misc]
     @requires_version(2, 13)
     def current_speed(self) -> int:
-        """Current speed of the heater-shaker's plate."""
+        """The current rpm of the Heater-Shaker's plate."""
         return self._module.speed
 
     @property  # type: ignore[misc]
     @requires_version(2, 13)
     def target_speed(self) -> Optional[int]:
-        """Target speed of the heater-shaker's plate."""
+        """The target rpm of the Heater-Shaker's plate."""
         return self._module.target_speed
 
     @property  # type: ignore[misc]
     @requires_version(2, 13)
     def temperature_status(self) -> str:
-        """Heater-shaker's temperature status string.
-
-        Returns one of these possible status values:
-        - "holding at target"
-        - "cooling"
-        - "heating"
-        - "idle"
-        - "error"
+        """One of five possible temperature statuses:
+
+        - ``holding at target``: The module has reached its target temperature and is actively maintaining that temperature.
+        - ``cooling``: The module has previously heated and is now passively cooling. `The Heater-Shaker does not have active cooling.`
+        - ``heating``: The module is heating to a target temperature.
+        - ``idle``: The module has not heated since the beginning of the protocol.
+        - ``error``: The temperature status can't be determined.
         """
         return self._module.temperature_status.value
 
     @property  # type: ignore[misc]
     @requires_version(2, 13)
     def speed_status(self) -> str:
-        """Heater-shaker's speed status string.
-
-        Returns one of these possible status values:
-        - "holding at target"
-        - "speeding up"
-        - "slowing down"
-        - "idle"
-        - "error"
+        """One of five possible shaking statuses:
+
+        - ``holding at target`` - The module has reached its target shake speed and is actively maintaining that speed.
+        - ``speeding up`` - The module is increasing its shake speed towards a target.
+        - ``slowing down`` - The module was previously shaking at a faster speed and is currently reducing its speed to a lower target or to deactivate.
+        - ``idle`` - The module is not shaking.
+        - ``error`` - The shaking status can't be determined.
```

Comment:
Use a `:` like the other changes here?
```suggestion
        - ``holding at target``: The module has reached its target shake speed and is actively maintaining that speed.
        - ``speeding up``: The module is increasing its shake speed towards a target.
        - ``slowing down``: The module was previously shaking at a faster speed and is currently reducing its speed to a lower target or to deactivate.
        - ``idle``: The module is not shaking.
        - ``error``: The shaking status can't be determined.
```